src/cache/cache_las.c

```diff
@@ -1462,40 +1478,64 @@ __wt_find_lookaside_upd(
          */
         if (upd_type == WT_UPDATE_MODIFY) {
             WT_NOT_READ(modify, true);
+            __find_birthmark_update(cbt, &birthmark_upd);
             while (upd_type == WT_UPDATE_MODIFY) {
```

Comment:
This loop exit in three conditions.
1. Whenever a UPDATE_STANDARD record type is found in LAS.
2. Whenever we found a LAS record of the same key that is less than a birthmark record.
3. Whenever we cross the key boundary.

I feel adding the above conditions in comments may be helpful.

Reply:
428f200